repos/fdbt-netex-output/src/netex-convertor/sharedHelpers.ts

```diff
@@ -449,14 +451,24 @@ export const getFareStructuresElements = (
 
         if ('productDuration' in product) {
             return [
-                getPeriodAvailabilityElement(availabilityElementId, validityParametersObject, hasTimeRestriction),
+                getPeriodAvailabilityElement(
+                    availabilityElementId,
+                    validityParametersObject,
+                    hasTimeRestriction,
+                    groupOfLinesRef,
+                ),
                 getDurationElement(ticket, product),
                 getPeriodConditionsElement(ticket, product),
             ];
         }
 
         return [
-            getPeriodAvailabilityElement(availabilityElementId, validityParametersObject, hasTimeRestriction),
+            getPeriodAvailabilityElement(
+                availabilityElementId,
+                validityParametersObject,
+                hasTimeRestriction,
+                groupOfLinesRef,
+            ),
```

Comment:
do we need to change the calling of function?

Reply:
It is being referenced in the function so if it is present it will use this value